app/components/ch/components/Table.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="table-container">
+    <table class="table">
+      <thead class="table__header">
+        <th class="table__header--entry">Name</th>
+        <th class="table__header--entry">Job</th>
+        <th class="table__header--entry">E-mail</th>
+        <th class="table__header--entry">Role</th>
+      </thead>
+      <tbody>
+        <tr class="table__row">
+          <td class="table__row--entry table__row--entry-highlight">
```

Comment:
you can use `<th>` in place of `<td>` here, and remove the `--entry-highlight` modifier. cf https://www.w3.org/WAI/tutorials/tables/two-headers/#table-with-an-offset-column-of-header-cells

Reply:
Changed with the switch to plain html elements